src/util.rs

```diff
@@ -1,22 +1,35 @@
+
+/// Type used for indexing entities.
```

Comment:
This crate is not necessarily used for an ECS (although `MAX_EID` looks a bit suspicious :smile:).